### DISCUSSION TITLE
Step pin for ATmega328

### DISCUSSION THREAD

**Discussion by MOCmaniac** [gin66/FastAccelStepper#209] [Q&A] [answered]
I'm doing a project at university where I'm using 3 steppers. Two nema 14 pancakes with a reduction of 7.5 and having to do 2 revolutions per second. I would like to use a microstepping of 8 or 16 (and have this microstepping interpolated up to 256 by a TMC2208). This gives 24000 pulses/second with a microstepping of 8. The third stepper must move at around 1 revolution per second and the microstepping can be much smaller (4 is sufficient -> 800 pulses/second).

I was planning to use an arduino nano for ease. I see that I can only use pins 9 and 10 with this library. Is it possible to connect the 3rd stepper to another pin as the frequency of the pulses is much smaller ?

Otherwise I will use an esp32.

Thanks in advance for your feedback

**Comment by gin66** [comment, marked as answer]
FastAccelStepper does not support that with the 328p, but you could either use AccelStepper or toggle the third stepper pins from app side with 800 pulses/s.

**Reply by MOCmaniac** [reply]
Thank you !
